src/learningPath/step/learningPathStepActions.js

```diff
@@ -11,6 +11,7 @@ import { routerActions } from 'react-router-redux';
 import get from 'lodash/get';
 
 import { applicationError, addMessage } from '../../messages/messagesActions';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Why do we need this?

Reply:
Because learningPathActions.js depends on learningPathStepActions.js and  learningPathStepActions.js and learningPathActions.js. It would take a major refactoring to fix the issue, which i beyond the scope of this PR. Either way I think the import/no-cycle is a good rule cause it has helped uncovering some badly structured code.